docs/Modders/Unique-parameters.md

```diff
@@ -31,7 +31,7 @@ This is because to the player, the text will be `non-Wounded Armor units`, which
 
 ## civFilter
 
-Allows filtering for specific civs.
+Allows filtering for specific civs. Contains [nationFilter](#nationfilter).
 
 - `All`
```

Comment:
Technically incorrect and unnecessary. We're repeating some stuff in civFilter that's actually in nationFilter. A short list here is fine for now